sql/core/src/test/scala/org/apache/spark/sql/execution/command/v1/CreateNamespaceSuite.scala

```diff
@@ -36,7 +36,7 @@ import org.apache.spark.sql.execution.command
 trait CreateNamespaceSuiteBase extends command.CreateNamespaceSuiteBase
     with command.TestsV1AndV2Commands {
   override def namespace: String = "db"
-  override def notFoundMsgPrefix: String = "Database"
+  override def notFoundMsgPrefix: String = if (runningV1Command) "Database" else "Namespace"
```

Comment:
We need to update the test now that v1 catalogs are run against both v1 and v2 commands (after adding `if conf.useV1Command`) .

Reply:
Done